website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -73,12 +73,12 @@ AfterOptions allows for one of two options to be displayed at a given time: foot
 The footer appends below the list displaying content and/or buttons (depending on the variant) that provide additional context (or functions) to the list options.
 
 ##### Single variant
-The footer defaults to hidden, but when enabled shows a counter (selected count / total count). Either of the two texts are togglable and customizable to fit context.
+The footer defaults to visible and shows a counter (selected count / total count). Either of the two texts are togglable and customizable to fit context.
```

Comment:
The selected count will always be 1 for the single select variant. What I currently have is displaying the number of results (e.g. 3 results)

Reply:
Oops, yeah we should remove selected count. My apologies and good catch.